code/__DEFINES/regex.dm

```diff
@@ -0,0 +1,2 @@
+/// Regex.Replace does not allow calling procs by name, but as of 515 these will always call the top level proc instead of overrides
```

Comment:
This doesn't document when you use REGEX_REPLACE_HANDLER (every time you use .Replace)